requirements/test.txt

```diff
@@ -2,23 +2,22 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    make upgrade
+#    pip-compile --output-file=requirements/test.txt requirements/test.in
```

Comment:
@magajh  Did you update this with `make upgrade`?

Reply:
Yes, this happened with make upgrade. Should I change this line to 'make upgrade' again?

Reply:
It is weird, `make upgrade` is configured to output this line with "make upgrade" in https://github.com/eduNEXT/eox-hooks/blob/0cab40e6174fbb14508fdc85f62c58d064178778/Makefile#L29

Reply:
you are right, I just checked and in my local repo I had this line commented this whole time 
`upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade`